### PR TITLE
Exchange info fix, server time option

### PR DESCRIPTION
**Pull request by megaherz**
- use server time option, handles `<APIError> code=-1021, msg=Timestamp for this request was 1000ms ahead of the server's time.`. Faced it often, most likely because distance between client-server. I'm in Belarus. 
- exchange info unmarshal fix. See https://github.com/adshao/go-binance/issues/53 for detail
- no need to handle close option 

There is an issue with CI. All tests passed with `go test ./...`

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -50,6 +50,11 @@ const (
 	recvWindowKey = "recvWindow"
 )
 
+var (
+	// UseServerTime enables getting server time to send in the timestamp key
+	UseServerTime = false
```
Comment (adshao) [inline client.go]: I think it is better to use RequestOption (https://github.com/adshao/go-binance/blob/master/client.go#L99) for this option

File: exchange_info_service.go [hunk 2]
```diff
@@ -45,5 +45,5 @@ type Symbol struct {
 	QuotePrecision     int                 `json:"quotePrecision"`
 	OrderTypes         []string            `json:"orderTypes"`
 	IcebergAllowed     bool                `json:"icebergAllowed"`
-	Filters            []map[string]string `json:"filters"`
+	Filters            []map[string]interface{} `json:"filters"`
```
Comment (adshao) [inline exchange_info_service.go]: move this change to another pr

Comment (cjamessutton) [reply exchange_info_service.go]: PR #59 contains this change.
